packages/repository/src/relations/has-many/has-many-through.repository.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Count, DataObject, Entity, Filter, Options, Where} from '../..';
+
+/**
+ * CRUD operations for a target repository of a HasManyThrough relation
+ */
```

Comment:
Let's add a warning that this interface is experimental. For example (based on [the spike pull request](https://github.com/strongloop/loopback-next/pull/2359/files#diff-177253a1afbdd8eb83b99efef86dd415R269-R271)):

```
 * EXPERIMENTAL: This interface is not stable and may change in the near future. 
 * Backwards-incompatible changes may be introduced in semver-minor versions.
```